src/eventListeners/colony.ts

```diff
@@ -96,5 +108,23 @@ export const setupListenersForColony = (colonyAddress: string): void => {
     addColonyEventListener(eventSignature, colonyAddress),
   );
 
-  addTokenEventListener(ContractEventsSignatures.Transfer, colonyAddress);
+  const tokenEvents = [
+    ContractEventsSignatures.Transfer,
+    ContractEventsSignatures.LogSetAuthority,
+  ];
+
+  /*
+   * @TODO once there are more relevant events in this list, we should extract this
+   * listener separately and run it for each token we see within our network
+   *
+   * Currently it works as we only care about one specific event which only happens
+   * on colony creation (Transfer doesn't count as it's being handled differently)
+   */
+  tokenEvents.forEach((eventSignature) => {
+    // This needs to be handled individually because transfer events are frequent and heavy
+    if (eventSignature === ContractEventsSignatures.Transfer) {
+      return addTokenTransferEventListener(eventSignature, colonyAddress);
+    }
+    return addTokenEventListener(eventSignature, tokenAddress);
+  });
```

Comment:
I agree with you that for now, this is overkill, however my point for having this in is to not have to deal with this in the future, when more token based events will be required. Just add them in, and this will do the rest.

I see both side of the argument as having validity here.